config/config_to_options.go

```diff
@@ -75,7 +76,7 @@ func (c *AppConfig) addFeeQuotes(options []engine.ClientOps) []engine.ClientOps
 	options = append(options, engine.WithFeeQuotes(c.ARC.UseFeeQuotes))
 
 	if c.ARC.FeeUnit != nil {
-		options = append(options, engine.WithFeeUnit(&utils.FeeUnit{
+		options = append(options, engine.WithFeeUnit(&bsv.FeeUnit{
```

Comment:
I've completely removed the `addFeeQuotes` function.

Reply:
I also removed this function